app/assets/stylesheets/views/_homepage.scss

```diff
@@ -251,3 +251,17 @@
     border-top: 4px solid $govuk-brand-colour;
   }
 }
+
+// Temporary overrides for global bar styling on homepage.
+// Doing this here avoids the need for republishing the
+// gem and therefore will be easier to remove later.
+.homepage.global-bar-present .global-bar {
```

Comment:
Is `.homepage.global-bar-present` necessary? Is `.global-bar-present` not specific enough?

Reply:
Great, thanks 👍